### PR TITLE
Fix ovsdb del interface

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: topology/probes/netlink/netlink.go [hunk 1]
```diff
@@ -842,7 +846,9 @@ func (u *NetNsProbe) onAddressDeleted(addr netlink.Addr, family int, index int64
 
 	intf := u.Graph.LookupFirstChild(u.Root, graph.Metadata{"IfIndex": index})
 	if intf == nil {
-		logging.GetLogger().Debugf("No interface with index %d for new address %s", index, addr.IPNet.String())
+		if _, err := u.handle.LinkByIndex(int(index)); err == nil {
+			logging.GetLogger().Errorf("Unable to find interface with index %d to del address %s", index, addr.IPNet.String())
+		}
```
Comment (hunchback) [inline topology/probes/netlink/netlink.go]: s/addr.IPNet.String()/addr.IPNet/

File: topology/probes/netlink/netlink.go [hunk 2]
```diff
@@ -812,7 +814,9 @@ func (u *NetNsProbe) onAddressAdded(addr netlink.Addr, family int, index int64)
 
 	intf := u.Graph.LookupFirstChild(u.Root, graph.Metadata{"IfIndex": index})
 	if intf == nil {
-		logging.GetLogger().Errorf("No interface with index %d for new address %s", index, addr.IPNet.String())
+		if _, err := u.handle.LinkByIndex(int(index)); err == nil {
+			logging.GetLogger().Errorf("Unable to find interface with index %d to add address %s", index, addr.IPNet.String())
```
Comment (hunchback) [inline topology/probes/netlink/netlink.go]: s/addr.IPNet.String()/addr.IPNet/

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Pull request merged**